packages/amplify-graphql-model-transformer/rds-lambda/package.json

```diff
@@ -10,7 +10,7 @@
     "babel-jest": "^29.1.2",
     "jest": "^29.1.2",
     "knex": "^2.3.0",
-    "mysql2": "^3.1.0",
+    "mysql2": "2.3.3",
```

Comment:
If the new version doesn't support that's alright, any particular reason we can't use the latest mysql2?

Reply:
There are some contract changes, I ran into some issues with types with 3.x. Also 2.x is stable and well-tested during POC.